app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= render partial: 'admin/header', locals: { name: "Application" } %>
+
+<h1><%= @application.applicant_name %></h1>
+<h3>Street Address: <%= @application.street_address %></h3>
+<h3>City: <%= @application.city %></h3>
+<h3>State: <%= @application.state %></h3>
+<h3>Zip Code: <%= @application.zip_code %></h3>
+<h3>Description: <%= @application.description %></h3>
+<h3>Pets on this Application:</h3>
+<table>
+  <thead>
+    <tr>
+      <th>Pet Name</th>
+      <th>Approve</th>
+      <th>Reject</th>
+      <th>Approval Status</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @application.pet_applications.each do |pet_application| %>
+      <tr>
+        <td><%= pet_application.pet.name %></td>
+        <td>
+          <% if pet_application.approval == nil && pet_application.pet.adoptable != false %> 
```

Comment:
Having a somewhat complex conditional decide whether you render a form is often the case. This is okay IMO and not `too complex`